tutorials/source-modeling/plot_beamformer_lcmv.py

```diff
@@ -197,6 +198,27 @@
 ###############################################################################
 stc.plot(mode='glass_brain', clim=dict(kind='value', lims=lims), **kwargs)
 
+###############################################################################
+# Morph the output to fsaverage
+#
+# We can also use volumetric morphing to get the data to fsaverage space. This
+# is for example necessary when comparing activity across subjects.
+# We pass a :class:`mne.SourceMorph` as the ``src`` argument to
+# `mne.VolSourceEstimate.plot`. To save some computational load when applying
+# the morph, we will crop the ``stc``:
+
+fetch_fsaverage(subjects_dir)  # ensure fsaverage src exists
+fname_fs_src = subjects_dir + '/fsaverage/bem/fsaverage-vol-5-src.fif'
+src_fs = mne.read_source_spaces(fname_fs_src)
+morph = mne.compute_source_morph(
+    forward['src'], subject_from='sample', src_to=src_fs,
+    subjects_dir=subjects_dir,
+    niter_sdr=[10, 10, 5], niter_affine=[10, 10, 5],  # just for speed
+    verbose=True)
+stc_fs = morph.apply(stc.copy().crop(0.05, 0.15))
```

Comment:
no need to `.copy()` here, since it's at the end of the tutorial and you don't need original `stc` again.

Reply:
Yep!